fairseq/models/distributed_fairseq_model.py

```diff
@@ -107,15 +107,14 @@ def DistributedFairseqModel(args, model, process_group, device):
                 args.slowmo_momentum = 0.5
             else:
                 args.slowmo_momentum = 0.6
+        slowmo_base_algorithm = SlowMoBaseAlgorithm[args.slowmo_base_algorithm.upper()]
 
-        wrapped_model = gossip.GossipDataParallel(
+        wrapped_model = SlowMoDistributedDataParallel(
```

Comment:
Should we test that there is parity between `SlowMoDistributedDataParallel` and `GossipDataParallel` before merging?

Reply:
Good point. I'll explain a bit more. The changes involved in converting `GossipDataParallel` to `SlowMoDistributedDataParallel` are all refactors with no inside logic being changed. I've tested on Slurm that the accuracies remain exactly the same after switching these APIs